emcc.py

```diff
@@ -38,6 +38,7 @@
 import time
 from subprocess import PIPE
 
+import emscripten
```

Comment:
might be worth doing the import in the only function that uses it? (it's not a tiny file, and we only need it when linking, not compiling.)

Reply:
Honestly I couldn't measure a difference and I think having imports all the the top is a lot cleaner.  I could be wrong here but I think it would premature optimization to do this kind of thing.